scout/server/blueprints/institutes/forms.py

```diff
@@ -12,6 +14,21 @@
 )
 from scout.constants import PHENOTYPE_GROUPS
 
+CASE_SEARCH_KEY = [
```

Comment:
Crash is this:
```  
File "/miniconda3/envs/py3.6/lib/python3.6/site-packages/wtforms/fields/core.py", line 454, in iter_choices
    for value, label in self.choices:
ValueError: too many values to unpack (expected 2)
```
Where self.choices = 
 
`['Case or individual name', 'HPO term', 'Search synopsis', 'Gene panel', 'Case status', 'Phenotype group', 'Patient cohort', 'Similar case', 'Similar phenotype', 'Pinned gene', 'Causative gene', 'Assigned user']`

Reply:
I propose a fix in `e371016`. WTForms documentation says that not-tuple list should work

> Select fields take a choices parameter which is a list of (value, label) pairs. It can also be a list of only values, in which case the value is used as the label. 

-but making (value, label) into the list solved the problem. 

(Local WTForm 2.2.1.)

Reply:
👍 Looks like a good solution, thanks!